packages/core-flows/src/customer/steps/delete-customers.ts

```diff
@@ -0,0 +1,30 @@
+import { ICustomerModuleService } from "@medusajs/types"
+import { createStep, StepResponse } from "@medusajs/workflows-sdk"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+
+type DeleteCustomerStepInput = string[]
+
+export const deleteCustomerStepId = "delete-customer"
+export const deleteCustomerStep = createStep(
+  deleteCustomerStepId,
```

Comment:
nit: Consistent with how other workflows and steps are named

```suggestion
export const deleteCustomersStepId = "delete-customer"
export const deleteCustomersStep = createStep(
  deleteCustomerStepId,
```